automation/components-functests.setup.sh

```diff
@@ -66,16 +66,16 @@ spec:
   kubeMacPool:
    rangeStart: "02:00:00:00:00:00"
    rangeEnd: "02:00:00:00:00:0F"
-  ovs: {}
-  macvtap: {}
-  kubeSecondaryDNS: {}
   kubevirtIpamController: {}
   imagePullPolicy: Always
 EOF
 
   if [[ $USE_KUBEVIRTCI == true ]]; then
     echo "  multus: {}" >> cr.yaml
     echo "  multusDynamicNetworks: {}" >> cr.yaml
+    echo "  ovs: {}" >> cr.yaml
+    echo "  macvtap: {}" >> cr.yaml
+    echo "  kubeSecondaryDNS: {}" >> cr.yaml
```

Comment:
why do we need this ?

Reply:
the fact it is here means dont deploy it on our lane
before this PR it deployed those, now it doesnt
so it free memory

Reply:
ok. Let's at least remove the linux-bridge. 

I wonder if we can fly without kubemacpool.

... but we can understand that later on.

Reply:
done